inductiva/fluids/post_processing/openfoam.py

```diff
@@ -231,11 +180,11 @@ def render_frame(self,
                      background_color: str = "white",
                      flow_cmap: str = "viridis",
                      object_color: str = "white",
-                     save_path: Path = None):
-        """Render flow property over the object in the WindTunnel."""
+                     save_path: types.Path = None):
+        """Render flow property over domain."""
```

Comment:
what do you mean by domain here?

Reply:
I refer on top, I do not want to repeat it everywhere.